README.md

```diff
@@ -49,10 +49,27 @@ conf.setNaaccrVersion("220");
 new MetafileTranslator().executeFullTranslation(conf);
 ```
 
+## Command Line
+The compiled JAR can be executed from the command line to perform a full translation.
+
+**Command Line Options:**
+- `-p`: The working folder containing the metafile
+- `-c`: The file name of the configuration properties file within the working folder
+
+**Example usage:**
+```sh
+$ java -jar validation-translation.jar -c config.properties -p WORKING_DIR
```

Comment:
I don't mean to be a pain about this pull request, sorry!

The CsvFieldResolver class makes sense, it looks great, we can add it.

I am still trying to figure out the rest. 

You are providing an example of how to run the CLI; have you tried that example yourself? It won't work because this library depends on other libraries, and the only way to make it a standalone JAR is to make a shadow JAR (or "fat" JAR, or whatever you want to call it).  Basically a JAR that contains all the classes of this library plus all the classes of the dependencies.  It's doable, but you will need to provide instructions on how to build that fat JAR, which would require a special Gradle task again.  

That makes the CLI pretty much unusable, unless it's embedded into another project. But if it's embedded, then you can simply copy its code into that project and not require a CLI class in this project.

So again, my question to you is: are you using the CLI class yourself, and if you are, how do you call it? From the command line, but you need dependencies. From an other project, but you could just have the CLI class in that project.

I guess what bothers me the most is that you are adding a dependency to a logging implementation.  It might not look like much, but it's rally not a great thing to do for a library that is supposed to abstract its logging via just an API and let the parent project deal with whatever implementation they want.  This little CLI class makes this project "not really a library", which is fine, except that IMO the CLI class is not useable as-is.   And I would prefer to not jump through hoops to make that work unless you are telling me that you need things to be this way.

My preference would be to only keep the CsvFieldResolver and nothing else; it's a great addition to the library.

If you feel strongly about keeping the CLI class, then I would like you to move it to the test package, and put back the logging implementation at the test level so it's not released.  You will also need to adjust the these instructions in the README file because those examples won't work as-is.

A third option is for you to move the CLI class and the examples in your other project where you plan on moving the compilation stuff that you removed earlier. 

Plenty of options, and we can certainly discuss this more if you don't agree with my point-of-view.

Reply:
No worries. Sorry for the delay, we are deep in 2024 changes...

I have run the CLI to translate the NCDB edits several times now, so I know it works! I originally modified the jar task to build a fat jar, but I have now split that off into a separate task, 'buildFatJar'. I will do some further work to move the CLI into the test package to prevent log4j from cluttering up the dependency tree.

Reply:
I agree it will work with a fat JAR. It wasn't clear to me that "validation-translation.jar" was supposed to be a fat JAR in your example:

```
$ java -jar validation-translation.jar -c config.properties -p WORKING_DIR
```

And that's kind of the whole argument for this; the full translation process (which includes creating the runtime classes) is really meant to be part of another project that takes care of compiling the runtime classes.  And if this project can't do the compilation, then why adding the rest (the CLI and the fat JAR)?   IMO, those two things should be part of your other project.  They are part of a "full translation" that requires another project setup. 

It's hard to explain. 

My preference would be for you to move the CLI and the fat JAR generation in your other project.  The only change that really makes sense for this library is the CSV field mapper. 

If you feel strongly about keeping the CLI and JAR generation in this project, that's fine, as long as log4j is not brought up into the released dependencies.  There have been too many security issues with it in the past few years...